packages/rpm/hazelcast.spec

```
@@ -47,8 +47,12 @@ rm -rf $RPM_BUILD_ROOT
 
 %files
 %{_prefix}/lib/%{name}/%{name}-%{hzversion}/*
-%config(noreplace) %{_prefix}/lib/%{name}/%{name}-%{hzversion}/bin/*.xml
-%config(noreplace) %{_prefix}/lib/%{name}/%{name}-%{hzversion}/bin/*.yaml
+%config(noreplace) %{_prefix}/lib/%{name}/%{name}-%{hzversion}/config/*.xml
+%config(noreplace) %{_prefix}/lib/%{name}/%{name}-%{hzversion}/config/*.yaml
+%config(noreplace) %{_prefix}/lib/%{name}/%{name}-%{hzversion}/config/*.options
+%config(noreplace) %{_prefix}/lib/%{name}/%{name}-%{hzversion}/config/*.properties
+%config(noreplace) %{_prefix}/lib/%{name}/%{name}-%{hzversion}/config/examples/*.yaml
+%config(noreplace) %{_prefix}/lib/%{name}/%{name}-%{hzversion}/config/examples/*.yaml
```

```suggestion
%config(noreplace) %{_prefix}/lib/%{name}/%{name}-%{hzversion}/config/examples/*.xml
```

it seems there is duplicate `*.yaml`